soc/arm/st_stm32/stm32l4/dts_fixup.h

```diff
@@ -335,4 +335,24 @@
 #define DT_LPTIM_1_CLOCK_BITS	DT_ST_STM32_TIMERS_40007C00_CLOCK_BITS_0
 #define DT_LPTIM_1_CLOCK_BUS	DT_ST_STM32_TIMERS_40007C00_CLOCK_BUS_0
 
+#define DT_DMA_1_BASE_ADDRESS	DT_ST_STM32_DMA_4002000_BASE_ADDRESS
+#define DT_DMA_1_IRQ		DT_ST_STM32_DMA_40020000_IRQ_0
+#define DT_DMA_1_IRQ_PRI	DT_ST_STM32_DMA_40020000_IRQ_0_PRIORITY
+#define DT_DMA_1_NAME		DT_ST_STM32_DMA_40020000_LABEL
+#define DT_DMA_1_CLOCK_BITS	DT_ST_STM32_DMA_40020000_CLOCK_BITS_0
+#define DT_DMA_1_CLOCK_BUS	DT_ST_STM32_DMA_40020000_CLOCK_BUS_0
+
+#define DT_DMA_2_BASE_ADDRESS	DT_ST_STM32_DMA_40020400_BASE_ADDRESS
+#define DT_DMA_2_IRQ		DT_ST_STM32_DMA_40020400_IRQ_0
+#define DT_DMA_2_IRQ_PRI	DT_ST_STM32_DMA_40020400_IRQ_0_PRIORITY
+#define DT_DMA_2_NAME		DT_ST_STM32_DMA_40020400_LABEL
+#define DT_DMA_2_CLOCK_BITS	DT_ST_STM32_DMA_40020400_CLOCK_BITS_0
+#define DT_DMA_2_CLOCK_BUS	DT_ST_STM32_DMA_40020400_CLOCK_BUS_0
+
+#define DT_DMAMUX_1_BASE_ADDRESS	DT_ST_STM32_DMAMUX_40020800_BASE_ADDRESS
+#define DT_DMAMUX_1_IRQ		DT_ST_STM32_DMAMUX_40020800_IRQ_0
+#define DT_DMAMUX_1_IRQ_PRI	DT_ST_STM32_DMAMUX_40020800_IRQ_0_PRIORITY
+#define DT_DMAMUX_1_NAME		DT_ST_STM32_DMAMUX_40020800_LABEL
+#define DT_DMAMUX_1_CLOCK_BITS	DT_ST_STM32_DMAMUX_40020800_CLOCK_BITS_0
+#define DT_DMAMUX_1_CLOCK_BUS	DT_ST_STM32_DMAMUX_40020800_CLOCK_BUS_0
```

Comment:
These are not needed since DMA instanciation code uses DT_INSTX_FOO defines

Reply:
removed